docker-compose.yml

```diff
@@ -132,6 +132,7 @@ services:
       SERVER_PORT: "8888"
       SPRING_CLOUD_CONFIG_SERVER_GIT_URI: "https://github.com/NDPH-ARTS/mts-trial-deployment-config.git"
       SPRING_CLOUD_CONFIG_SERVER_GIT_SEARCHPATHS: "sample_trial/services, sample_trial/services/*"
+      EUREKA_CLIENT_SERVICEURL_DEFAULTZONE: "http://discovery-server:8761/eureka/"
```

Comment:
Is this working? I think it will override the local profile which disables the discovery, since env vars come before.

Reply:
it works. the defaultzone is used IF discovery is enabled (which for local is not true)